demisto_sdk/commands/init/initiator.py

```diff
@@ -599,13 +797,45 @@ def ignore_secrets(self, secrets):
                 "[yellow]Could not find the .secrets-ignore file - make sure your path is correct[/yellow]"
             )
 
+    def verify_output_path_for_xsiam_content(self) -> bool:
+        """Verify that there is a output path from the user
+
+        Returns:
+            bool. True if all the required inputs from the user are valid.
+        """
+        if self.xsiam and not self.output:
```

Comment:
What happens here is I give only --xsiam? which means creating the whole pack.
We should check the output for --intergtaion -- xsiam only